webapp/client/src/stories/DisplayFailureIcon.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import DisplayFailureIcon from "../components/DisplayFailureIcon";
+
+export default {
+  title: "Anchor Elements/DisplayFailureIcon",
```

Comment:
This should be in the Form Fields section or?

Reply:
What do you mean? What should be part where? The title?

Reply:
The section in the story book. Your Icon is under the Anchor Elements

Reply:
Oh! Now i get it! Maybe "Forms" section is better than "Form Fields" section?

Reply:
Oh yes sry Forms :D

Reply:
Done :)